lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_update_metadata.rb

```diff
@@ -16,11 +16,11 @@ def self.run(params)
       #####################################################
 
       def self.description
-        'Downloads translated metadata from the translation system'
+        'Downloads translated metadata from the translation system (deprecated)'
       end
 
       def self.details
-        'Downloads translated metadata from the translation system'
+        'Downloads translated metadata from the translation system. This action is deprecated in favor of android_download_translations(…)'
```

Comment:
Is the trailing `(…)` intentional? If not: 

```suggestion
        'Downloads translated metadata from the translation system. This action is deprecated in favor of android_download_translations'
```